pytket/binders/circuit/Circuit/add_classical_op.cpp

```diff
@@ -237,20 +237,24 @@ void init_circuit_add_classical_op(
           "add_c_setreg",
           [](Circuit &circ, const _tket_uint_t value, const BitRegister &reg,
              const py::kwargs &kwargs) {
+	    if (value >> reg.size() != 0) {
```

Comment:
It appears that when `reg.size() >=32` the shift amount is reduced mod 32, so we actually want
```suggestion
	    if (reg.size() < 32 && (value >> reg.size()) != 0) {
```

Reply:
Yes, this is in the last commit.